### PR TITLE
[FIX] Make bi-sided clustering default

### PR DESCRIPTION
**Pull request by tsalo**
Closes None.

Changes proposed in this pull request:

- Make the default thresholding method for cluster-extent thresholding "bi-sided" instead of "two-sided". This will cluster positively and negatively weighted voxels separately. Currently, they are merged into super-clusters.

### CODE REVIEW

**Comment by jbteves** [comment]
@tsalo is this a WIP?

**Comment by tsalo** [comment]
No, I just forgot to switch branches when I made that commit.

**Comment by jbteves** [comment]
I hate it when I do that. Re-approved.

**Comment by tsalo** [comment]
@emdupre @handwerkerd any conceptual concerns about this change?

**Comment by emdupre** [comment]
What is the effect here on the outcomes ?

**Comment by tsalo** [comment]
Any impact is minimal. In practice (using the five-echo test dataset), I get the same values for most metrics, except for small differences in `countnoise` (higher with the proposed changes, as some mixed "clusters" are no longer counted as clusters).

**Comment by dowdlelt** [comment]
Conceptually this makes sense to me - in that if there is a alternating pattern of positive and negative betas on a map this would group that into a super cluster, when it is more likely to be an artifact from motion/GRAPPA, that is, reflective of bad structure in the data. So I am pro this.

**Comment by jbteves** [comment]
@dowdlelt do you mind reading over the code and formally reviewing it?

File: tedana/utils.py [hunk 1]
```diff
@@ -250,10 +250,10 @@ def threshold_map(img, min_cluster_size, threshold=None, mask=None,
         Boolean array for masking resultant data array. Default is None.
     binarize : bool, optional
         Default is True.
-    sided : {'two', 'one', 'bi'}, optional
+    sided : {'bi', 'two', 'one'}, optional
         How to apply thresholding. One-sided thresholds on the positive side.
         Two-sided thresholds positive and negative values together. Bi-sided
-        thresholds positive and negative values separately. Default is 'two'.
+        thresholds positive and negative values separately. Default is 'bi'.
     """
```
Comment (dowdlelt) [inline tedana/utils.py]: Should this section specify the details of what is returned?

Comment (tsalo) [reply tedana/utils.py]: Added.

**Comment by jbteves** [comment]
@all-contributors add @dowdlelt for review

**Comment by tsalo** [comment]
Thanks @dowdlelt and @jbteves! Merging now.

**Pull request merged**